pages/linux/du.md

```diff
@@ -1,23 +1,27 @@
 # du
 
-> Estimate file space usage.
+> Disk usage: estimate and summarize file and folder space usage.
 
-- Get a sum of the total size of a file/folder in human readable units:
+- List the sizes of a folder and any subfolders, in the given unit (B/KB/MB):
 
-`du -sh {{file_or_directory}}`
+`du -{{b|k|m}} {{path/to/folder}}`
 
-- List file sizes of a directory and any subdirectories in KB:
+- List the sizes of a folder and any subfolders, in human-readable form (i.e. auto-selecting the appropriate unit for each size):
 
-`du -k {{file_or_directory}}`
+`du -h {{path/to/folder}}`
 
-- List file sizes of a directory and any subdirectories in MB:
+- Show the size of a single folder, in human readable units:
 
-`du -m {{file_or_directory}}`
+`du -sh {{path/to/folder}}`
 
-- Get recursively, individual file/folder sizes in human readable form:
+- List the human-readable sizes of a folder and of all the files and folders within it (recursively):
```

Comment:
It is not entirely redundant since "folder and of all the files and folders within it" could be interpreted as the first level only, but yeah, I'm not opposed to removing it since it's not a big deal anyway. I'll remove since we have more to gain from the simplicity than from perfect explicitness in this case.